components/ReferenceManager/references/reference_uploader/ReferenceManualUploadDrawer.tsx

```diff
@@ -117,7 +114,12 @@ export default function ReferenceManualUploadDrawer(): ReactElement {
             required: referenceSchemaValueSet.required,
           });
         };
-        if (schemaField === "pdf_url" || schemaField === "custom") {
+        console.log(schemaField);
```

Comment:
nit: should probably remove the console.log

Reply:
Ah yes, def should remove